xarray/tests/test_accessor_dt.py

```diff
@@ -248,7 +248,7 @@ def test_dask_accessor_method(self, method, parameters) -> None:
         assert_equal(actual.compute(), expected.compute())
 
     def test_seasons(self) -> None:
-        dates = pd.date_range(start="2000/01/01", freq="M", periods=12)
+        dates = pd.date_range(start="2000/01/01", freq="ME", periods=12)
```

Comment:
(not sure what we want to do with this failure on old versions of pandas; probably OK to skip this, but defer to @spencerkclark. And no need to delay this PR)